Sources/Generation/Generation.swift

```diff
@@ -102,4 +89,18 @@ public extension Generation {
         
         return tokenizer.decode(tokens: output)
     }
+
+    private func logitsWrappers(config: GenerationConfig) -> [any LogitsWarper] {
```

Comment:
```suggestion
    private func logitsWarpers(config: GenerationConfig) -> [any LogitsWarper] {
```

Let's make sure we always use the right nomenclature `LogitsWarper` in all classes and variable names :)